src/brackets.js

```diff
@@ -162,50 +153,50 @@ define(function (require, exports, module) {
         // in the modules since they would run in context of the unit test window,
         // and would not have access to the app html/css.
         brackets.test = {
-            CodeHintManager         : CodeHintManager,
-            CodeInspection          : CodeInspection,
-            CommandManager          : CommandManager,
-            Commands                : Commands,
+            CodeHintManager         : require("editor/CodeHintManager"),
```

Comment:
I removed the require above since we should only require module over there that we actually need in our code, for example for calling their methods.
The only occurence of CodeHintManager, though, is here in `brackets.test`.

See #1783 (case 2) for more details.